### PR TITLE
Remove Ratable characteristic

### PR DESCRIPTION
**Pull request by libertymayc**
Remove ratable characteristic; tokens removed:

- --uitk-ratable-cursor-hover
- --uitk-ratable-cursor-active
- --uitk-ratable-cursor-undo
- --uitk-ratable-cursor-disabled
- --uitk-ratable-borderWidth
- --uitk-ratable-borderWidth-undo
- --uitk-ratable-borderStyle
- --uitk-ratable-borderStyle-undo
- --uitk-ratable-background
- --uitk-ratable-background-active
- --uitk-ratable-background-activeDisabled
- --uitk-ratable-background-hover
- --uitk-ratable-background-undo
- --uitk-ratable-borderColor
- --uitk-ratable-borderColor-active
- --uitk-ratable-borderColor-disabled
- --uitk-ratable-borderColor-hover
- --uitk-ratable-borderColor-undo

Leading to removal of rate tokens from palette:

- --uitk-palette-rate-background
- --uitk-palette-rate-background-active
- --uitk-palette-rate-background-activeDisabled
- --uitk-palette-rate-background-hover
- --uitk-palette-rate-background-undo
- --uitk-palette-rate-border
- --uitk-palette-rate-border-active
- --uitk-palette-rate-border-disabled
- --uitk-palette-rate-border-hover
- --uitk-palette-rate-border-undo

Add new tokens to measured characteristic for components to be restyled:

- --uitk-measured-foreground-hover
- --uitk-measured-foreground-active
- --uitk-measured-foreground-undo
- --uitk-measured-foreground-activeDisabled
- --uitk-palette-measured-foreground-active
- --uitk-palette-measured-foreground-activeDisabled

### CODE REVIEW

File: .changeset/dry-turtles-sit.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@jpmorganchase/uitk-lab": minor
+"@jpmorganchase/uitk-theme": minor
+---
+
+Remove ratable characteristic; add --uitk-measured-foreground-hover, --uitk-measured-foreground-active, --uitk-measured-foreground-undo, --uitk-measured-foreground-activeDisabled, --uitk-palette-measured-foreground-active, --uitk-palette-measured-foreground-activeDisabled
```
Comment (joshwooding) [inline .changeset/dry-turtles-sit.md:6]: I find this style of changeset hard to read. What do you think of using something like:
```suggestion
Remove ratable characteristic
Add new tokens to measured characteristic:
- --uitk-measured-foreground-hover
- --uitk-measured-foreground-active
- --uitk-measured-foreground-undo
- --uitk-measured-foreground-activeDisabled
- --uitk-palette-measured-foreground-active
- --uitk-palette-measured-foreground-activeDisabled
```

Comment (joshwooding) [reply .changeset/dry-turtles-sit.md:6]: Probably worth listing out the ratable tokens that have been removed

**Review by joshwooding — CHANGES_REQUESTED** [review]
See comment

File: .changeset/dry-turtles-sit.md (hunk 1 above)
Comment (libertymayc) [reply .changeset/dry-turtles-sit.md:6]: haha agreed, will change now, it's because i did it quickly and lazily in the terminal :P

**Review by joshwooding — DISMISSED** [review]

**Review by joshwooding — APPROVED** [review]

**Review by joshwooding — APPROVED** [review]

**Pull request merged**